### DISCUSSION TITLE
Problema con Docker Desktop

### DISCUSSION THREAD

**Discussion by JuaniLezcano** [frlp-utn-ingsoft/catedra#105] [Q&A] [answered]
Todas las veces que instale Docker Desktop en mi PC de escritorio me comienzan los crasheos de Windows, queria saber si alguno tiene idea de alguna solución o si no me queda otra que formatear.

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/107371324/99e0140f-0aff-486a-ab9d-0ce06430f4db)

**Comment by rodriguezemautn** [comment, marked as answer]
Podrias verificar en el setup de tu bios que la opcion "Virtualization technology" este seteado en "enabled"

Es probable que ese sea el problema. También puedes verificar los requisitos y recomendaciones en la documentación oficial: [enlace a la documentación de Docker para instalación en Windows](https://docs.docker.com/desktop/install/windows-install/).

Otra opción es instalar una máquina virtual en VirtualBox o en el hipervisor que prefieras, con Debian sin interfaz gráfica. Solo necesitas el servidor SSH para realizar la práctica.
